src/staff-administrator/components/staff-add-edit-user/staff-user-check-answers/staff-user-check-answers.component.ts

```diff
@@ -65,8 +67,13 @@ export class StaffUserCheckAnswersComponent implements OnInit {
         type: InfoMessageType.SUCCESS
       } as InformationMessage);
       this.router.navigateByUrl('/staff', { state: { retainMessages: true } });
-    }, () => {
-      this.router.navigateByUrl('/service-down');
+    }, (err) => {
+      if (err && err?.status === 400) {
+        this.errMsg = err;
```

Comment:
Can you add some checks in here for blank error messages etc. Something like
if (!?err.error?.errorDescription) {
  this.errMsg = { error: { errorDescription: "Your user creation request could not be processed: }}
}
Otherwise the use is not going to get anything shown at the top, but the request won't have worked